apps/glusterfs/volume_entry.go

```diff
@@ -304,6 +304,11 @@ func (v *VolumeEntry) Create(db *bolt.DB,
 			break
 		}
 	}
+
+	if err != nil {
+		return err
+	}
+
```

Comment:
You will also need to make sure unit tests are adjust to expect the correct paths to test the change